_posts/2018-11-12-sr2019-rules-released.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: news
+title: SR2019 Rules Published & Useful Resources
+---
+
+We had a successful Kickstart on Saturday and it was great to see the excited teams getting familiar with our kit.
+
+Our game this year is called Caldera in which teams need to claim the most zones by placing more of their tokens than the other robots within the zone boundaries. ([full rules][rules])
+
+We’ve already seen the teams come up with some innovative solutions and it’s going to be interesting to see how they develop.
+
+All competitors should now have user accounts for our website. This includes our [IDE][ide] for teams to write their code in. This also includes [our forums][forums] which are a great place to brainstorm with other teams and discuss any problems which you run into.
+
+(See [our docs][account-info] if you are not able to log in.)
+
+If you weren’t able to attend Kickstart this year, or would like a recap, you can download:
+- [The presentation][presentation] in which we explain this year’s game and how the competition is running this year.
+- [The microgames][microgames] which are small activities designed to let you become familiar with the Student Robotics kit.
+
+[rules]: {{ site.baseurl }}/docs/rules/
```

Comment:
I have a feeling that using `{{ site.baseurl }}` doesn't work consistently for things outside this repo. I can't recall exactly what the case was, but there are some gotchas around it. I suggest having a look at how we link to the IDE elsewhere (for example the header) and copying that pattern.

Reply:
The header just links to the page with `href="/ide"` So I have put the full links instead of `{{ site.baseurl }}`.